wrapper.py

```diff
@@ -273,22 +276,17 @@ def _load_dataset(f, lazy=False):
         return None
     else:
         if hasattr(f, 'value'):
-            if str(f.value) == 'None':
+            if f.attrs['_value_type'] == 'NoneType':
```

Comment:
Why is this `NoneType` instead of `None`? did you also change that in the `save` function?